querybook/webapp/redux/search/action.ts

```diff
@@ -137,13 +137,24 @@ export function performSearch(): ThunkResult<Promise<ISearchPreview[]>> {
                     });
                     break;
                 case SearchType.Table:
-                    searchRequest = SearchTableResource.search({
-                        ...searchParams,
-                        metastore_id:
-                            state.dataTableSearch.metastoreId ||
-                            queryMetastoresSelector(state)[0].id,
-                        fields: Object.keys(searchState.searchFields),
-                    });
+                    if (isVectorSearch) {
+                        searchRequest = SearchTableResource.vectorSearch({
+                            metastore_id:
+                                state.dataTableSearch.metastoreId ||
```

Comment:
state.dataTableSearch.metastoreId ||
                                queryMetastoresSelector(state)[0].id
---
turn this into a var